books/workshops/references/bibtex-parser.lisp

```diff
@@ -0,0 +1,399 @@
+; ACL2 BibTeX Parser
+; Parses BibTeX files and creates alist representations
+
+; Aakash Koneru
```

Comment:
Can you add copyright+licensing information to the header (and for the other file)? See any of the books in the `kestrel` directory for what that looks like. Copyright Kestrel and license should be the ACL2 top-level license.